deploy/006-deploy-pool-registry.ts

```diff
@@ -15,20 +15,30 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   );
   const proxyOwnerAddress = await toAddress(preconfiguredAddresses.NormalTimelock || "account:deployer", hre);
 
+  const defaultProxyAdmin = await hre.artifacts.readArtifact(
+    "hardhat-deploy/solc_0.8/openzeppelin/proxy/transparent/ProxyAdmin.sol:ProxyAdmin",
+  );
+
```

Comment:
I would add a comment mentioning why we are doing this

Reply:
[2837377](https://github.com/VenusProtocol/isolated-pools/pull/423/commits/2837377e9893ca5b18fccf6116314e3fd8f81478)